pkg/handler/eval.go

```diff
@@ -169,13 +169,34 @@ var logEvalResult = func(r *models.EvalResult, dataRecordsEnabled bool) {
 		rateLimitPerFlagConsoleLogging(r)
 	}
 
+	logEvalResultToDatadog(r)
+
 	if !config.Config.RecorderEnabled || !dataRecordsEnabled {
 		return
 	}
 	rec := GetDataRecorder()
 	rec.AsyncRecord(r)
 }
 
+var logEvalResultToDatadog = func(r *models.EvalResult) {
+	if config.Global.StatsdClient == nil {
+		return
+	}
+
+	config.Global.StatsdClient.Incr(
+		"evaluation",
+		[]string{
+			fmt.Sprintf("EntityType:%s", util.SafeStringWithDefault(r.EvalContext.EntityType, "null")),
+			fmt.Sprintf("FlagID:%d", util.SafeUint(r.FlagID)),
+			fmt.Sprintf("FlagSnapshotID:%d", util.SafeUint(r.FlagSnapshotID)),
```

Comment:
is this necessary?
generally, we should use relative bucket value (such as 1d-ago, 2d-ago), rather than absolute value.